templates/compose/open-archiver.yml

```
@@ -0,0 +1,77 @@
+# documentation: https://openarchiver.com
+# slogan: A self-hosted, open-source email archiving solution with full-text search capability.
+# tags: email archiving,email,compliance,search
+# logo: svgs/openarchiver.svg
+# port: 3000
+
+services:
+    open-archiver:
+        image: logiclabshq/open-archiver:latest
```

Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.

```suggestion
        image: logiclabshq/open-archiver:v0.3.4
```